cas-server-core-api-ticket/src/main/java/org/jasig/cas/ticket/TicketState.java

```diff
@@ -29,14 +31,14 @@
      * @return the previous time used.
      */
 
-    long getPreviousTimeUsed();
+    ZonedDateTime getPreviousTimeUsed();
 
     /**
      * Get the time the ticket was created.
      *
      * @return the creation time of the ticket.
      */
-    long getCreationTime();
+    ZonedDateTime getCreationTime();
```

Comment:
Don't think this will work with JPA ticket registries. We'll likely run into either schema or serialization issues. Let's keep this as long.